build-tools/automation/azure-pipelines.yaml

```
@@ -454,7 +454,7 @@ stages:
         testName: Mono.Android_TestsMultiDex
         project: tests/Runtime-MultiDex/Mono.Android-TestsMultiDex.csproj
         testResultsFiles: TestResult-Mono.Android_TestsMultiDex-$(ApkTestConfiguration).xml
-        artifactName: Mono.Android_Tests-Signed.apk
+        artifactName: Mono.Android-TestsMultiDex-Signed.apk
```

```suggestion
        artifactName: Mono.Android_TestsMultiDex-Signed.apk
```
@jonpryor it's swap `-` to `_`!